src/components/ec-toaster/ec-toaster.spec.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-use-before-define */
+import { mount } from '@vue/test-utils';
+import EcToaster from './ec-toaster.vue';
+
+const messages = [
+  {
+    id: 1,
+    type: 'success',
+    title: 'This is the title',
+    subtitle: 'This is the subtitle',
+    dismissable: true,
+  },
+];
+
+function mountToaster(props, mountOpts) {
+  return mount(EcToaster, {
+    propsData: { ...props },
+    ...mountOpts,
+  });
+}
+
+describe('EcToaster', () => {
+  describe('messages prop', () => {
+    it('should not render the toaster if no props are passed', () => {
+      const wrapper = mountToaster();
+      expect(wrapper.find('.ec-toaster').exists()).toBe(false);
+      expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('should render a toaster if the messages prop is passed', () => {
+      const wrapper = mountToaster({ messages });
+
+      expect(wrapper.find('.ec-toaster').exists()).toBe(true);
+      expect(wrapper.find('.ec-alert').exists()).toBe(true);
+      expect(wrapper.find('.ec-alert').classes('ec-alert--success')).toBe(true);
+      expect(wrapper.find('.ec-alert__title').text()).toBe('This is the title');
+      expect(wrapper.find('.ec-alert__subtitle').text()).toBe('This is the subtitle');
+      expect(wrapper.find('.ec-alert__dismiss-icon').exists()).toBe(true);
```

Comment:
You can verify all of these in the generated snapshot. I think it is enough for the test.